.github/workflows/ci.yml

```diff
@@ -39,8 +47,18 @@ jobs:
           - nextest run -p cairo-lang-syntax
           - nextest run -p cairo-lang-syntax-codegen
           - nextest run -p cairo-lang-test-runner
+          - nextest run -p cairo-lang-test-runner --features serde
           - nextest run -p cairo-lang-test-utils
           - nextest run -p cairo-lang-utils
+          - nextest run -p cairo-lang-utils --no-default-features
+          - nextest run -p cairo-lang-utils --no-default-features --features parity-scale-codec
+          - nextest run -p cairo-lang-utils --no-default-features --features serde
+          - nextest run -p cairo-lang-utils --no-default-features --all-features
+          - nextest run -p cairo-lang-vm-utils
+          - nextest run -p cairo-lang-vm-utils --no-default-features
+          - nextest run -p cairo-lang-casm-contract-class
+          - nextest run -p cairo-lang-casm-contract-class --no-default-features
+          - nextest run -p cairo-lang-casm-contract-class --no-default-features --features serde
```

Comment:
Is all this really required?
Can't you just do a run with no_std + supported features and another with all features including std?